packages/react/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,33 +1,292 @@
 import {clsx} from 'clsx'
 import type {To} from 'history'
-import React from 'react'
+import React, {useState, useRef, useCallback, useEffect, useMemo} from 'react'
 import type {SxProp} from '../sx'
 import type {ComponentProps} from '../utils/types'
 import classes from './Breadcrumbs.module.css'
 import type {ForwardRefComponent as PolymorphicForwardRefComponent} from '../utils/polymorphic'
 import {BoxWithFallback} from '../internal/components/BoxWithFallback'
+import {ActionMenu} from '../ActionMenu'
+import {ActionList} from '../ActionList'
+import {IconButton} from '../Button/IconButton'
+import {KebabHorizontalIcon} from '@primer/octicons-react'
+import {useResizeObserver} from '../hooks/useResizeObserver'
+import type {ResizeObserverEntry} from '../hooks/useResizeObserver'
 
 const SELECTED_CLASS = 'selected'
 
 export type BreadcrumbsProps = React.PropsWithChildren<
   {
     className?: string
+    overflow?: 'wrap' | 'menu'
+    hideRoot?: boolean
```

Comment:
Definitely not sure what the right names would be, but what would you think of something like:

```suggestion
    overflow?: 'wrap' | 'hide' | 'hide-root'
```

So that we wouldn't have the case of `hideRoot` only making sense with `overflow="menu"`. It would also avoid the confusion here with it not being a `menu` since this would be a popup with a list of links versus `role="menu"`

Reply:
Am I correct in saying that hideRoot would only be relevant with overflow: `menu`? If yes, how do you feel about `overflow: 'menu' | 'menu-with-hidden-root'`?